DcCore/DcCore/DC/events.swift

```diff
@@ -27,17 +27,17 @@ public class DcEventHandler {
         let accountId = event.accountId
 
         if event.id >= DC_EVENT_ERROR && event.id <= 499 {
-            logger.error("📡[\(accountId)] \(event.data2String)")
+            logger.error("[\(accountId)] \(event.data2String)")
             return
         }
 
         switch event.id {
 
         case DC_EVENT_INFO:
-            logger.info("📡[\(accountId)] \(event.data2String)")
+            logger.info("[\(accountId)] \(event.data2String)")
 
         case DC_EVENT_WARNING:
-            logger.warning("📡[\(accountId)] \(event.data2String)")
+            logger.warning("[\(accountId)] \(event.data2String)")
 
         case DC_EVENT_CONFIGURE_PROGRESS:
             logger.info("📡[\(accountId)] configure: \(Int(data1))")
```

Comment:
```suggestion
            logger.info("[\(accountId)] configure: \(Int(data1))")
```
if you want to remove the icon everywhere